mpc-core/src/point_addition/mod.rs

```diff
@@ -5,50 +5,51 @@
 //! would also be possible using this approach, but it hasn't been
 //! implemented here).
 
-pub mod errors;
-pub mod master;
-pub mod slave;
+mod follower;
+mod leader;
 
-pub use crate::{
-    msgs::point_addition::PointAdditionMessage, point_addition::errors::PointAdditionError,
-};
+use std::{convert::TryFrom, ops::Add};
+
+pub use crate::msgs::point_addition::PointAdditionMessage;
 use curv::{arithmetic::Converter, BigInt};
-pub use master::PointAdditionMaster;
-pub use slave::PointAdditionSlave;
+pub use follower::{state as follower_state, PointAdditionFollower};
+pub use leader::{state as leader_state, PointAdditionLeader};
 
 /// NIST P-256 Prime
 pub const P: &str = "ffffffff00000001000000000000000000000000ffffffffffffffffffffffff";
 
-pub type SecretShare = BigInt;
+/// Additive secret share of a NIST P-256 private key
```

Comment:
This should read:
/// Additive secret share of the resulting X coordinate.

And also below it should be referred to as "share" rather than "key"